apps/site/test/site_web/views/trip_plan_view_test.exs

```diff
@@ -1132,14 +1132,14 @@ closest arrival to 12:00 AM, Thursday, January 1st."
 
     test "gets fare by type" do
       non_transit_leg = @itinerary.legs |> List.first()
-      assert get_fare_by_type(non_transit_leg, :highest_one_way_fare) == nil
-      assert get_fare_by_type(non_transit_leg, :lowest_one_way_fare) == nil
-      assert get_fare_by_type(non_transit_leg, :reduced_one_way_fare) == nil
+      assert Fares.get_fare_by_type(non_transit_leg, :highest_one_way_fare) == nil
```

Comment:
Could you please also just move these tests to `fares_test.exs` to match the implementation?